books.js

```diff
@@ -41,9 +160,10 @@ function addBook(Title, Author) {
 }
 
 function removeList(name, author) {
-  for (let x = 0; x < bookList.length; x += 1) {
-    if ((bookList[x].title === name) && (bookList[x].author === author)) {
-      bookList.splice(x, 1);
+  const arr = lists.printArray();
+  for (let x = 0; x < arr.length; x += 1) {
+    if ((arr[x].title === name) && (arr[x].author === author)) {
+      lists.remove(x);
       break;
     }
   }
```

Comment:
- Good job adding the class 👍 however according to the requirements you should Create class methods to add and remove books. right now normal functions are handling these requests, please change that.

Reply:
This issue is well resolved! ✔️